include/oneapi/mkl/rng/detail/hiplike_helper.hpp

```diff
@@ -0,0 +1,208 @@
+ * documentation and/or other materials provided with the distribution.
+ *
+ * (3) Neither the name of the University of California, Lawrence Berkeley
+ * National Laboratory, U.S. Dept. of Energy nor the names of its contributors
+ * may be used to endorse or promote products derived from this software
+ * without specific prior written permission.
+ *
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ * You are under no obligation whatsoever to provide any bug fixes, patches,
+ * or upgrades to the features, functionality or performance of the source
+ * code ("Enhancements") to anyone; however, if you choose to make your
+ * Enhancements available either publicly, or directly to Lawrence Berkeley
+ * National Laboratory, without imposing a separate written license agreement
+ * for such Enhancements, then you hereby grant the following license: a
+ * non-exclusive, royalty-free perpetual license to install, use, modify,
+ * prepare derivative works, incorporate into other computer software,
+ * distribute, and sublicense such enhancements or derivative works thereof,
+ * in binary and source code form.
+ *
+ * If you have questions about your rights to use or distribute this software,
+ * please contact Berkeley Lab's Intellectual Property Office at
+ * IPO@lbl.gov.
+ *
+ * NOTICE.  This Software was developed under funding from the U.S. Department
+ * of Energy and the U.S. Government consequently retains certain rights.  As
+ * such, the U.S. Government has been granted for itself and others acting on
+ * its behalf a paid-up, nonexclusive, irrevocable, worldwide license in the
+ * Software to reproduce, distribute copies to the public, prepare derivative
+ * works, and perform publicly and display publicly, and to permit others to do
+ * so.
+ ******************************************************************************/
+#ifndef _MKL_RNG_HIPLIKE_HELPER_HPP_
+#define _MKL_RNG_HIPLIKE_HELPER_HPP_
+
+namespace oneapi {
+namespace mkl {
+namespace rng {
+
+// Static template functions oneapi::mkl::rng::curand::range_transform_fp for
+// Buffer and USM APIs
+//
+// cuRAND has no built-in functionality to specify a custom range for sampling
```

Comment:
Indeed this file is common. however, it should not be anymore since we are not touching the cuRAND backend. 
* Removed `hiplike_helper.hpp`
* Included functions into `rocrand_helper.hpp`
Tests still pass with the new change:
[rocrng_ct_tests_still_pass.txt](https://github.com/oneapi-src/oneMKL/files/8655214/rocrng_ct_tests_still_pass.txt)